python_wrapper/setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='twa',
-    version='0.0.7',
+    version='0.0.8-paper',
```

Comment:
Please remove the `-paper` tag from the version before merging.

Reply:
Addressed in https://github.com/TheWorldAvatar/baselib/pull/31/commits/cfcc8cce114fff05dcec5845678d450cd272ac36